Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -265,6 +266,8 @@ MainWindow::MainWindow(Core::System& system, std::unique_ptr<BootParameters> boo
           &MainWindow::ShowAchievementSettings);
   connect(m_game_list, &GameList::OpenAchievementSettings, this,
           &MainWindow::ShowAchievementSettings);
+  connect(&Settings::Instance(), &Settings::HardcoreStateChanged, this,
+          &MainWindow::OnHardcoreChanged);
```

Comment:
You are listening to Settings::HardcoreStateChanged, but there is no longer any code that emits Settings::HardcoreStateChanged.

Reply:
Okay, now there's code that emits Settings::HardcoreStateChanged, but what is the purpose? Aren't you already catching everything using just the config changed callback?